src/index.css

```diff
@@ -1,7 +1,276 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --pale-gray: #f5f6fa;
+  --white-dark: #fafafa;
+  --white-light: #fff;
+  --body-back: rgb(237, 234, 234);
+  --blue-azure: #0290ff;
+  --gray-dark: #888;
+  --black-dark: #121212;
+}
+
+h1 {
+  font-size: x-large;
+}
+
 body {
   font-family: 'Montserrat', sans-serif;
+  background-color: var(--pale-gray);
+  height: 100%;
+}
```

Comment:
- [ ] Kindly go through your project and ensure that you are using all the correct font-family throughout
- [ ] Please ensure that your background color is the same as the given design